README.md

```diff
@@ -76,6 +76,10 @@ TODO: create k8s cluster and test!
 
 ## Troubleshooting and FAQ
 
+### nvidia-docker 
```

Comment:
A couple of thoughts here : 

- The content of this section does not follow the FAQ pattern of the readme, All of the sections begin with questions. 
- It closely resembles the nvidia-docker blurb in Prerequisites (but without my guide), Please work to further differentiate it from the prereqs blurb. 
- The url for `(here)[https://github.com/NVIDIA/nvidia-docker]` is not displaying correctly as you have the brackets backward, (here)[https://github.com/NVIDIA/nvidia-docker] vs [here](https://github.com/NVIDIA/nvidia-docker)

Reply:
Thanks for the suggestions, and sorry about the stupid mistake re: url.
I think my latest commit nails it down; please have a look.